Scripts/travis.sh

```diff
@@ -61,12 +66,14 @@ execute_xcodebuild() {
 }
 
 if [ "${TYPE}" == "RUBY" ]; then
-  rvm use 2.2.2;
+  rvm use 2.2.5;
   cd gem;
   bundle install --retry=3;
   rake;
 elif [ "${TYPE}" == "UNIT" ]; then
   execute_xcodebuild Tests/UnitTests/UnitTests.xcodeproj EarlGreyUnitTests
+elif [ "${TYPE}" == "FUNCTIONAL" && "${FILTER}" != "" ]; then
```

Comment:
After working with this for a bit, I reverted the script changes since as long as Xcode 8 doesn't support iOS 8 tests, we are not going to use the filter in our tests.